core/src/main/java/org/elasticsearch/action/admin/cluster/node/tasks/list/TaskInfo.java

```diff
@@ -132,9 +159,18 @@ public void writeTo(StreamOutput out) throws IOException {
         } else {
             out.writeBoolean(false);
         }
+        out.writeLong(startTime);
+        out.writeLong(runningTime);
         parentTaskId.writeTo(out);
     }
 
+    public static final class Fields {
+        static final XContentBuilderString START_TIME = new XContentBuilderString("start_time");
```

Comment:
I believe we've been just using the string version of field instead of these lately.